app/rsr/controllers/getPredictorScores.js

```diff
@@ -1,17 +1,25 @@
 const { Controller } = require('hmpo-form-wizard')
+const { getDraftPredictorScore } = require('../../../common/data/hmppsAssessmentApi')
+const logger = require('../../../common/logging/logger')
+const { splitPredictorScores } = require('../../../common/utils/predictors')
 
 class GetPredictorScores extends Controller {
-  locals(req, res, next) {
-    res.locals.predictorScores = {
-      current: {
-        date: '13 Aug 2021 at 08:26:21',
-        scores: {
-          RSR: { type: 'RSR', level: 'HIGH', score: 11.34 },
-          OSPC: { type: 'OSP/C', level: 'MEDIUM', score: 8.76 },
-          OSPI: { type: 'OSP/I', level: 'LOW', score: 3.45 },
-        },
-      },
-    }
+  // eslint-disable-next-line consistent-return
+  async locals(req, res, next) {
```

Comment:
what's the standard regarding testing controllers in the fronted? i can see some controllers have tests but many of them don't? is that something we should add to the new rsr stuff to avoid regression maybe?

Reply:
We definitely do need to add in some tests - was planning to spend the rest of today working some of those up